code-scanning/properties/eslint.properties.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "ESLint",
+    "creator": "ESLint",
+    "description": "A tool for identifying and reporting the problems found in ECMAScript/JavaScript code.",
+    "iconName": "eslint",
+    "categories": [
+        "Code Scanning",
+        "javascript",
```

Comment:
should we add typescript?

Reply:
Added

Reply:
For consistency please follow the Pascal naming convention as used in other workflows: "JavaScript", "TypeScript", "EcmaScript"